### ISSUE TITLE
add git as a dependency for node bundling

### ISSUE THREAD

**Issue by benjamintd** [mapbox/stork#33]
According to this documentation, git URLs are valid npm dependencies in a `package.json` file: https://docs.npmjs.com/files/package.json#git-urls-as-dependencies

However, stork errors with this message: 
```
[Container] 2017/12/21 15:16:02 Running command npm install --production
npm ERR! code ENOGIT
npm ERR! No git binary found in $PATH
npm ERR! Failed using git.
npm ERR! Please check if you have git installed and in your PATH.
```

Would it make sense to include git as a dependency in the node dockerfiles? Or should I use a custom `buildspec.yaml` in that situation?

**Comment by rclark** [comment]
> Would it make sense to include git as a dependency in the node dockerfiles?

:+1: sounds good to me. That'd be a welcome PR.

**Comment by vsmart** [comment]
@benjamintd Can this be closed - was this issue fully adressed in #34?

**Comment by benjamintd** [comment]
Yes, closing the issue, thanks!
